### ISSUE TITLE
Server Folders

### ISSUE THREAD

**Issue by LandenStephenss** [SyntheticGenerations/Orcinus#4]
You should put all commands in their own folders per category, so that ./commands doesnt look like complete garbage

**Comment by Katrixerse** [comment]
That is true I do this with my current bots now but never updated this one to do so, I'll look at adding it, thanks for the suggestion.
